easybuild/easyconfigs/p/PETSc/PETSc-3.7.5-intel-2016a-download.eb

```diff
@@ -0,0 +1,41 @@
+name = 'PETSc'
+version = '3.7.5'
```

Comment:
include some `versionsuffix` to make it clear this is using mostly 'included' dependencies rather than ones provided through EB?